tests/robot/tests/4_edi_multi_versioned_models/4.0_check_edi_multi_versioned_models.robot

```diff
@@ -166,4 +166,46 @@ Check EDI model inspect by valid model id and invalid version
     [Tags]  edi  cli  enclave  multi_versions
     ${resp}=   Run EDI inspect by model id and model version    ${MODEL_TEST_ENCLAVE}    ${TEST_MODEL_ID}   ${TEST_MODEL_1_VERSION}test
                     Should Be Equal As Integers                 ${resp.rc}          0
-                    Should be empty                             ${resp.stdout}
\ No newline at end of file
+                    Should be empty                             ${resp.stdout}
+
+Check EDI scale up all instances for 2 models(diff versions) by versions=*
+    [Documentation]  Try to scale up 2 models with different versions but the same id by all versions through EDI console
+    [Tags]  edi  cli  enclave  multi_versions
+    ${resp}=        Run EDI scale with version      ${MODEL_TEST_ENCLAVE}   ${TEST_MODEL_ID}    2   '*'
+                    Should Be Equal As Integers     ${resp.rc}              0
+    ${resp}=        Run EDI inspect with parse by model id       ${MODEL_TEST_ENCLAVE}      ${TEST_MODEL_ID}
+    ${model_1}=     Find model information in edi   ${resp}      ${TEST_MODEL_ID}   ${TEST_MODEL_1_VERSION}
+                    Log                             ${model_1}
+    ${model_2}=     Find model information in edi   ${resp}      ${TEST_MODEL_ID}   ${TEST_MODEL_2_VERSION}
+                    Log                             ${model_2}
+                    Verify model info from edi      ${model_1}   ${TEST_MODEL_ID}   ${TEST_MODEL_IMAGE_1}   ${TEST_MODEL_1_VERSION}  2
+                    Verify model info from edi      ${model_2}   ${TEST_MODEL_ID}   ${TEST_MODEL_IMAGE_2}   ${TEST_MODEL_2_VERSION}  2
+
+Check EDI scale up all instances for 2 models(diff versions) by id=*
+    [Documentation]  Try to scale up 2 models with different versions but the same id by all ids through EDI console
+    [Tags]  edi  cli  enclave  multi_versions
+    ${resp}=        Run EDI scale                   ${MODEL_TEST_ENCLAVE}   '*'    2
```

Comment:
We should use asterisk only for model version in test because * in model ID may affect other tests

Reply:
@kirillmakhonin so the only tests with asterisk that will make no side affect to other ones are inspect '*'. Maybe we should leave only them and other ones (undeploy, scale) add in mock unit tests?